lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -319,22 +320,24 @@ <h4 class="message-title">${_('Almost there!')}</h4>
               <a href="${reverse('verify_student_verify_now', kwargs={'course_id': unicode(course_overview.id)})}" class="btn" data-course-id="${course_overview.id}">${_('Verify Now')}</a>
             </div>
           % elif verification_status['status'] == VERIFY_STATUS_SUBMITTED:
-            <h4 class="message-title">${_('You have already verified your ID!')}</h4>
-            <p class="message-copy">${_('Thanks for your patience as we process your request.')}</p>
+            <h4 class="message-title">${_('You have submitted your verification information.')}</h4>
+            <p class="message-copy">${_('You will see a message on your dashboard when the verification process is complete (usually within 1-2 days).')}</p>
+          % elif verification_status['status'] == VERIFY_STATUS_RESUBMITTED:
+            <h4 class="message-title">${_('You have submitted your reverification information.')}</h4>
```

Comment:
@tasawernawaz Is this a mistake? I thought the title @srpearce provided for this message is `Your current verification will expire soon!`